sites/public/src/layouts/application.tsx

```diff
@@ -1,18 +1,138 @@
-import React from "react"
+import React, { useContext } from "react"
 import { useRouter } from "next/router"
 import Head from "next/head"
-import { getSiteFooter, getSiteHeader } from "../lib/helpers"
-import { t } from "@bloom-housing/ui-components"
+import { MenuLink, setSiteAlertMessage, t } from "@bloom-housing/ui-components"
+import { AuthContext } from "@bloom-housing/shared-helpers"
+import { getSiteFooter } from "../lib/helpers"
+import { SiteHeader } from "@bloom-housing/doorway-ui-components/src/headers/SiteHeader"
 
 const Layout = (props) => {
+  const { profile, signOut } = useContext(AuthContext)
+
   const router = useRouter()
+
+  const languages =
+    router?.locales?.map((item) => ({
+      prefix: item === "en" ? "" : item,
+      label: t(`languages.${item}`),
+    })) || []
+
+  const menuLinks: MenuLink[] = [
+    {
+      title: t("pageTitle.welcome"),
+      href: "/",
+    },
+    {
+      title: t("nav.viewListings"),
+      href: "/listings",
+    },
+    {
+      title: t("nav.helpCenter"),
+      href: "#",
+      subMenuLinks: [
+        {
+          title: t("pageTitle.getStarted"),
+          href: "/help/get-started",
+        },
+        {
+          title: t("pageTitle.housingHelp"),
+          href: "/help/housing-help",
+        },
+        {
+          title: t("pageTitle.questions"),
+          href: "/help/questions",
+        },
+      ],
+    },
+  ]
+  if (process.env.housingCounselorServiceUrl) {
+    menuLinks.push({
+      title: t("pageTitle.getAssistance"),
+      href: process.env.housingCounselorServiceUrl,
+    })
+  }
+  if (process.env.showProfessionalPartners) {
+    menuLinks.push({
+      title: t("nav.professionalPartners"),
+      href: "#",
+      subMenuLinks: [
+        {
+          title: t("pageTitle.developersAndPropertyManagers"),
+          href: "/professional-partners/developers-and-property-managers",
+        },
+        {
+          title: t("pageTitle.jurisdictions"),
+          href: "/professional-partners/jurisdictions",
+        },
+      ],
+    })
+  }
+  if (profile) {
```

Comment:
This isn't specific to this, but it definitely exposes it more. When you shrink the screen with all of these navigations there is some weird behavior. As you shrink the screen the navigation links drop down below the site logo and stay there until they eventually jump into the hamburger menu. However, when you re-grow the screen the nav links reappear below the logo and stay there no matter how big the screen gets
![image](https://github.com/metrotranscom/doorway/assets/42942267/f55451b5-869f-4d9e-a065-52c83349b890)

Reply:
Believe I fixed this @ludtkemorgan !